README.md

```diff
@@ -38,7 +38,70 @@ npm install
 
 ## Usage
 
-TBD
+### Encoding Verifiable Presentations as QR Code Images
+
+```js
+import vpqr from '@digitalbazaar/vpqr';
+
+const vp = {
+  "@context": "https://www.w3.org/2018/credentials/v1",
+  "type": "VerifiablePresentation",
+  "verifiableCredential": {
+    "@context": [
+      "https://www.w3.org/2018/credentials/v1",
+      "https://w3id.org/cit/v1",
+      "https://w3id.org/security/suites/ed25519-2020/v1"
+    ],
+    "id": "urn:uuid:188e8450-269e-11eb-b545-d3692cf35398",
+    "type": [
+      "VerifiableCredential",
+      "ConcealedIdTokenCredential"
+    ],
+    "issuer": "did:key:z6Mku2yHBpMvm13KqtupN7MNEj7Y7AREUWsFfGBRhyP9YZf7",
+    "issuanceDate": "2021-03-24T20:03:03Z",
+    "expirationDate": "2021-06-24T20:03:03Z",
+    "credentialSubject": {
+      "concealedIdToken": "zo58FV8vqzY2ZqLT4fSaVhe7CsdBKsUikBMbKridqSyc7LceLmgWcNTeHm2gfvgjuNjrVif1G2A5EKx2eyNkSu5ZBc6gNnjF8ZkV3P8dPrX8o46SF"
+    },
+    "proof": {
+      "type": "Ed25519Signature2020",
+      "created": "2021-03-24T20:03:03Z",
+      "verificationMethod": "did:key:z6Mku2yHBpMvm13KqtupN7MNEj7Y7AREUWsFfGBRhyP9YZf7#z6Mku2yHBpMvm13KqtupN7MNEj7Y7AREUWsFfGBRhyP9YZf7",
+      "proofPurpose": "assertionMethod",
+      "proofValue": "z5Z33Pi9zWUWMHdxhG1Ko7ssgeHsuT3ocyg46pJTTjbwk1YjjtSCa4CQ5UyjnkQ6ar5ohY5Kv2mncWbfgEqWcVrTQ"
+    }
+  }
+};
+
+const documentLoader = {/*...*/}; // Set up your documentLoader
```

Comment:
Move comment line above to keep a consistent format.

Reply:
Done, thanks.